src/form-generator/form-handler-api-web-forms.ts

```diff
@@ -0,0 +1,118 @@
+import { FormElementCreation } from "../form-element-creator-strategy/form-element-creator";
+import { FormValidator } from "../form-validator/validator";
+import { HtmlTagName } from "../html-tag-name";
+import { FormSuccessMessage } from "../form-element-creator/form-success-message";
+import { FormRequiredFieldsParagraph } from "../form-element-creator/form-required-fields-paragraph";
+import { DivCreatorWithClassName } from "../form-element-creator/div-creator";
+import { classNames } from "../config/class-name";
+import { config } from "../config/config-attributes";
+import { ButtonCreation } from "../form-element-creator/button-creation";
+
+export class FormHandlerApiWebForms {
+  private formElementCreation: FormElementCreation;
+  private formValidator: FormValidator;
+  private formSuccessMessage: FormSuccessMessage;
+  private formRequiredFieldsParagraph: FormRequiredFieldsParagraph;
+
+  constructor() {
+    this.formElementCreation = new FormElementCreation();
+    this.formValidator = new FormValidator();
+    this.formSuccessMessage = new FormSuccessMessage();
+    this.formRequiredFieldsParagraph = new FormRequiredFieldsParagraph();
+  }
+
+  createFormElement = async (): Promise<HTMLDivElement> => {
+    const container = new DivCreatorWithClassName().createDiv(
+      classNames.container
+    );
+    const title = new DivCreatorWithClassName().createDiv(
+      classNames.titleOneStep
+    );
+    title.textContent = config.titleOneStep.textContent;
+
+    const form = new HtmlTagName().formCreator();
+    form.id = "contact-form";
+
+    const elements = [...config.selects, ...config.fields];
+
+    for (const element of elements) {
+      if (element.name !== "Password" && element.name !== "Confirm Password") {
+        await this.formElementCreation.create(form, element);
+      }
+    }
```

Comment:
lepiej odfiltrowac sobie te pola na starcie. A potem wykonac ta metode. 
Najlepiej w ogole tego fora wydzielic do nowej metody bo metoda zewnetrzna nazywa sie createFormElement a w srodku jakis X creatow leci . To bardziej ta metoda powinna nazywac sie createFormElements. 
`for (const element of elements) {
      if (element.name !== "Password" && element.name !== "Confirm Password") {
        await this.formElementCreation.create(form, element);
      }
    }`
To moze byc oddzielna metoda ale tak jak mowie mozna sobie uzyc elements.filter( bla bla ) 
bedize lepiej